docs/guides/basics/parameters.md

```diff
@@ -0,0 +1,251 @@
+
+
+## Suggested values
+
+As a convenience to users of your Pack, you can provide a suggested value for a parameter. When they use your formula the default will be pre-populated in the formula editor, action dialog, etc. The user is then free to edit or replace it this value.
+
+To add a suggested value to a parameter set the field `defaultValue` to the value you'd like to use. The suggested value must be of the same type as the parameter, for example a number parameter must have a number as it's suggested default value.
+
+```ts
+coda.makeParameter({
+  type: coda.ParameterType.Number,
+  name: "days",
+  description: "How many days of data to fetch.",
+  defaultValue: 30,
+})
+```
+
+Currently suggested values are only used for required parameters, and setting them for optional parameters has no effect. To
+
+
+## Accepting multiple values
+
+For some formulas you may want to allow the user to enter multiple values for a parameter. You could use an array parameter for this case but a more user-friendly approach may be to use variable argument (vararg) parameters. These are parameters that you allow the user to repeat as many times as needed.
+
+```
+Foo(List("A", "B", "C"))  # A string array parameter.
+Foo("A", "B", "C")        # A string variable argument parameter.
+```
+
+They are defined using the `varargParameters` property and accept the same parameter objects. The values set by the user are passed in to the `execute` just like normal parameters, only their is an unknown number of them. The easiest way to access them is by using [JavaScript's "rest" syntax][mdn_rest], which captures the remaining values into an array.
+
+```ts
+pack.addFormula({
+  // ...
+  parameters: [
+    coda.makeParameter({
+      type: coda.ParameterType.String,
+      name: "name",
+      description: "The person's name.",
+    }),
+  ],
+  varargParameters: [
+    coda.makeParameter({
+      type: coda.ParameterType.String,
+      name: "nickname",
+      description: "A nickname for the person.",
+    }),
+  ],
+  // ...
+  execute: async function ([name, ...nicknames], context) {
+    // ...
+  },
+});
+```
+
+There some important difference between about vararg parameters and standard parameters:
+
+- They appear at the end of the formula, after all standard parameters.
+- Unlike standard parameters they are optional by default, and cannot by made required.
+- You can't provide a default value, since the user must always enter an explicit value.
+- You can have more than one, but if so the user is required to enter complete sets of values. For example, if you have two vararg parameters `a` and `b`, the user can't provide a value for `a` without also providing a value for `b`. These pairs of parameters can then be repeated multiple times: `Foo("a1", "b1", "a2", "b2")`.
+
+```ts
+--8<-- "examples/formula/steps.ts"
```

Comment:
I think this code sample maybe didn't get rendered properly? It's blank in the staged files.

Reply:
Same cause and fix as above.